diff --git a/includes/menu.inc b/includes/menu.inc
index 2be0903..9c33482 100644
--- a/includes/menu.inc
+++ b/includes/menu.inc
@@ -1856,7 +1856,7 @@ function menu_navigation_links($menu_name, $level = 0) {
   $router_item = menu_get_item();
   $links = array();
   foreach ($tree as $item) {
-    if (!$item['link']['hidden']) {
+    if ($item['link']['access'] && !$item['link']['hidden']) {
       $class = '';
       $l = $item['link']['localized_options'];
       $l['href'] = $item['link']['href'];
